README.md

```diff
@@ -4,12 +4,25 @@
 ![Codecov](https://img.shields.io/codecov/c/github/keptn/lifecycle-toolkit?token=KPGfrBb2sA)
 ![goversion](https://img.shields.io/github/go-mod/go-version/keptn/lifecycle-toolkit?filename=operator%2Fgo.mod)
 ![version](https://img.shields.io/github/v/release/keptn/lifecycle-toolkit)
-![status](https://img.shields.io/badge/status-not--for--production-red)
 [![GitHub Discussions](https://img.shields.io/github/discussions/keptn/lifecycle-toolkit)](https://github.com/keptn/lifecycle-toolkit/discussions)
 
 The goal of this toolkit is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.
 It is an incubating project, under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+Status of the different features:
+
+- ![status](https://img.shields.io/badge/status-stable-brightgreen) Observability
+- ![status](https://img.shields.io/badge/status-alpha-orange) Metrics
+- ![status](https://img.shields.io/badge/status-alpha-orange) Release lifecycle
```

Comment:
I would suggest putting this into a table and also adding a column for the API group URL.
so something like
```
| Name | API Group | Stability Status |
| ---- | --------- | ---------------- |
| ... | ... | ... |
```

Reply:
Also,  we don't have any observability APIs :)
we have lifecycle, metrics and options as API groups

Reply:
Exactly, that's why I am calling features. Things like context propagation or runtime support are not really bound to an API group. 
However, I think also adding a table with the API group and their maturity makes sense 🤔

Reply:
Aha ok! Then i misunderstood. But I agree, the same thing for the APIs would be cool :)